### ISSUE TITLE
Bintray links are down

### ISSUE THREAD

**Issue by RiQuY** [Starman0620/re3#1]
Bintray links are down, 404 page shows up when trying to download from any of them.

I suggest choosing another website to upload or offer a torrent/magnet to avoid DMCA takedowns.

**Comment by RiQuY** [comment]
Possibly the repo will be takedown too so, ¿what about GitLab?

**Comment by ViperAcidZX** [comment]
Fuck Rockstar and fuck Take-Two.

**Comment by Starman0620** [comment]
Yeah I'm not the original creator of any of this, I simply uploaded because I hate Take Two with a dying passion. Have ever since the whole OIV LC incident in 2017

**Comment by NodokaHanamura** [comment]
Thanks for this. I've gone ahead and cloned it locally for archival. Rockstar and T2 can go burn in hell.

**Comment by LdieSU7** [comment]
ipfs?

**Comment by JonasNo** [comment]
Too bad you don't have the history for the source code. Kicking myself for not cloning the repo when I first read about it.
